client/src/reducers/DirectoryReducer.js

```diff
@@ -21,6 +21,13 @@ const DirectoryReducer = (state = initialState, action) => {
       newState = Object.assign({}, state, { loading: false });
       newState[action.role] = action.users;
       return newState;
+    case 'GET_OUTLOOK_USERS_SUCCESS':
+      const { outlookUsers } = action;
+      console.log(outlookUsers)
```

Comment:
Maybe we could add a debug condition or just remove this logging to reduce log spam